tests/wicked/before_test.pm

```diff
@@ -64,8 +64,7 @@ sub run {
     } else {
         # Common SUT Configuration
         if (get_var('WICKED_SOURCES')) {
-            zypper_call('--quiet in -t pattern Basis-Devel');
```

Comment:
why not install the pattern anymore? I did not see it covered in the autoyast profile

Reply:
I failed to find pattern in SLE 15 SP2 and while trying to solve this somehow I figure out that actually I don't need a pattern but only two packages make and gcc . And even if I will find pattern in SLE 15 SP2 ( what I plan to do anyway ) I still prefer to keep it this way ( I checked that it working for SLE 12 SP5 too )

Reply:
Personally I prefer having all required packages listed explicitly as this is not obscuring potential problems in the future.
If we want to test if given pattern installs expected packages, we could have slim and inexpensive test for that (or we perhaps have that already for this pattern).